code/game/objects/structures/votingbox.dm

```diff
@@ -17,6 +17,7 @@
 	var/vote_description = ""
 
 	var/list/voted //List of ID's that already voted.
+	var/cooldown //To prevent paper spam
```

Comment:
use COOLDOWN_DECLARE and the other cooldown defines

Reply:
Right, yeah, there's a thing for that now.